test/README.md

```diff
@@ -0,0 +1,92 @@
+# Integration tests
+
+This directory contains a test suite that can be run to validate conduit
+functionality via a series of end-to-end tests.
+
+## Prerequisites
+
+The integration tests operate on your currently configured Kubernetes cluster.
+Prior to running the test suite, verify that:
+
+- The conduit images you're trying to test have been built and are accessible
+  to the Kubernetes cluster to which you are deploying
+- The kubectl CLI has been configured to talk to that Kubernetes cluster
+- The conduit namespace that you're testing does not already exist
+- The repo's go dependencies have been downloaded by running `bin/dep ensure`
+
+## Running tests
+
+You can use the `bin/test-run` script to run the full suite of tests.
+
+The `bin/test-run` script requires an absolute path to a conduit binary to test
+as the first argument. You can optionally pass the namespace where conduit will
+be installed as the second argument.
+
+For instance, to test your installed version of conduit in the "specialtest"
+namespace, from the root of the conduit repo, run:
+
+```bash
+$ bin/test-run `which conduit` specialtest
+```
+
+That will create multiple namespaces in your Kubernetes cluster:
+
+```bash
+$ kubectl get ns | grep specialtest
+specialtest               Active    4m
+specialtest-egress-test   Active    2m
+specialtest-get-test      Active    1m
+...
+```
+
+You can also test a locally-built version of conduit.
+
+For instance, to test your current branch on minikube, first build all of the
+conduit images in your minikube environment, by running:
+
+```bash
+$ DOCKER_TRACE=1 bin/mkube bin/docker-build
+```
+
+That command also outputs the corresponding conduit binaries into the
+`target/cli` directory, and you can use the `bin/conduit` script to load those
+binaries when running tests. From the root of the conduit repo, run:
+
+```bash
+$ bin/test-run `pwd`/bin/conduit
```

Comment:
related to my comment at the top, perhaps some mention of getting docker images into the environment that is to be tested? as written i think these instructions will only work in minikube.

Reply:
There's a bullet point about this in the Prerequisites section of this doc:

```
- The conduit images you're trying to test have been built and are accessible
  to the Kubernetes cluster to which you are deploying
```

I can call that out more explicitly though, and will add a note about supported environments.